### PR TITLE
fix: Ontologies displayed twice (DEV-1325)

### PR DESCRIPTION
**Pull request by EricSommerhalder**
resolves DEV-1325

### CODE REVIEW

File: src/app/project/ontology/ontology.component.ts [hunk 1]
```diff
@@ -382,7 +383,26 @@ export class OntologyComponent implements OnInit {
             (ontologies: ReadOntology[]) => {
                 // update current list of project ontologies
                 ontologies[ontologies.findIndex(onto => onto.id === ontology.id)] = ontology;
-                this._cache.set('currentProjectOntologies', ontologies);
+                // avoid duplicates
+                const temp: ReadOntology[] = [];
```
Comment (mdelez) [inline src/app/project/ontology/ontology.component.ts:387]: I would rename these temp variables to something a bit more meaningful

File: src/app/project/ontology/ontology.component.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import { O } from '@angular/cdk/keycodes';
```
Comment (mdelez) [inline src/app/project/ontology/ontology.component.ts:1]: auto-import FTW?

**Review by mdelez — CHANGES_REQUESTED** [review]

File: src/app/project/ontology/ontology.component.ts (hunk 2 above)
Comment (EricSommerhalder) [reply src/app/project/ontology/ontology.component.ts:1]: got to love it

**Review by mdelez — APPROVED** [review]
works well, thanks :)

**Pull request merged**